returnn/datasets/sprint.py

```diff
@@ -1235,6 +1235,10 @@ def supports_seq_order_sorting(self) -> bool:
         """supports sorting"""
         return True
 
+    def supports_sharding(self) -> bool:
+        """:return: whether this dataset supports sharding"""
+        return True
```

Comment:
```suggestion
        return False
```
Or not?

Reply:
This is the `SprintCacheDataset`, which reads sprint cache files w/ RETURNN's own implementation, I think that can do it. At least it uses `get_seq_order_for_epoch`.

Reply:
Ah sorry, I did not see that in the diff.